docs/bis/ec/ec-format.md

```diff
@@ -14,6 +14,8 @@
 - `fractional`
 - `scientific`
 - `station`
+- `bearing`
```

Comment:
we have to be careful here ... this is not supported in EC at this point.  When it is we will need to identify that it's only available in a new EC version ... I guess 3.3

Reply:
I added a note saying this is only supported in the quantity package at the moment